incubator/exporter-otlp.spec/src/main/scripts/io/aklivity/zilla/specs/exporter/otlp/schema/otlp.schema.patch.json

```diff
@@ -0,0 +1,66 @@
+[
```

Comment:
Is this configuration forwards compatible for the future use of gRPC instead of JSON for OTLP protocol?